assets/common/components/register_model/spec.yaml

```diff
@@ -1,6 +1,6 @@
 $schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
 name: register_model
-version: 0.0.8
+version: 0.0.8.reg2
```

Comment:
lot of temp changes. Could you please remove them?

Reply:
done